test/tools/fuzzTesting/fuzzHelper.cpp

```diff
@@ -257,32 +257,36 @@ std::string RandomCodeBase::generate(int _maxOpNumber, RandomCodeOptions const&
 		eth::Instruction inst = (eth::Instruction) opcode;
 		eth::InstructionInfo info = eth::instructionInfo(inst);
 
-		if (info.name.find("INVALID_INSTRUCTION") != std::string::npos || info.name.empty()
+		while (info.name.find("INVALID_INSTRUCTION") != std::string::npos || info.name.empty()
```

Comment:
Why is this a loop now, what is it iterating over?  What changes between the first run and the second run?

Reply:
random codes are genereated in for sycle. 
if a random code is undefined  then we change it to a defined opcode in a while loop untill it is defined. (if we dont accept undefined opcodes)

there was an issue with static seed. a random undefined opcode would lead to endless loop.